### DISCUSSION TITLE
My aedt-icepak seems to be stuck with thousands of monitors

### DISCUSSION THREAD

**Discussion by YLSHA** [ansys/pyaedt#4351] [Q&A] [answered]
In my case, I performed a thermal simulation on a 1mm*1mm*0.5mm block, and when only a few monitors were placed, the simulation could be completed, and the results could be output in just a few minutes. 
However, after I added 4851 evenly placed monitors, the simulation took 60+ minutes and didn't end, as shown in the following figure on the GUI interface.
![image](https://github.com/ansys/pyaedt/assets/139062983/c111f2c1-27b0-411e-ab17-59ff80bbb4b3)
![image](https://github.com/ansys/pyaedt/assets/139062983/3f274df6-ed9a-47d9-9885-aa18266c927c)

 I also didn't find the file mentioned in the message in the project folder. 
Dose Someone knows what went wrong. Thank you very much!

**Comment by lorenzovecchietti** [comment, marked as answer]
Hi @YLSHA, if the simulation is not completed, than the problem is not related to pyAEDT. Please try to start the analysis from the GUI, most probably you will find the same result. If so, please create a support request through https://www.ansys.com/support.
